google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectServiceTest.java

```diff
@@ -184,6 +187,43 @@ public void testGetServiceNameFromAppEngineWebXml_serviceAndModule() {
                 "testData/descriptor/appengine-web_runtime-java8-serviceandmodule.xml")));
   }
 
+  public void testGenerateAppYaml() {
+    Path outputDir = Paths.get(getProject().getBasePath() + "/src/main/appengine");
+    appEngineProjectService.generateAppYaml(
```

Comment:
just wondering - in these test cases are the files temporary (and cleaned up after execution)?